packages/gatsby-theme-aio/algolia/query-builder.js

```diff
@@ -116,7 +116,11 @@ class QueryBuilder {
       };
       records = await this.createRecordsForFrame.execute(node, options);
     } else if (node.openAPISpec) {
-      const options = {};
+      const options = {
+        tempDir: './public/bootprint',
```

Comment:
What happens if there's a dir named `/static/bootprint/some_files` which will be copied into `public/bootprint`, will it collide with this tempDir and break the build ? If yes, it would make sense to use a hash to name the tempDir to avoid any user defined folders wdyt ?

Reply:
I agree and will change it soon on a future update.